frontend/app/model/Eventbrite.scala

```diff
@@ -179,7 +190,7 @@ object Eventbrite {
   }
 
   case class DiscountBenefitTicketing(generalRelease: EBTicketClass, member: EBTicketClass) {
-    val saving = generalRelease.priceInPence - member.priceInPence
+    val saving = (generalRelease.priceInPence + generalRelease.feeInPence) - member.priceInPence
```

Comment:
I agree with you there @chrisjowen, relying on operator precedence makes things harder to read in my opinion